app/pages/lab/project-details.cjsx

```diff
@@ -126,7 +126,8 @@ module.exports = React.createClass
 
           <p>
             <AutoSave tag="label" resource={@props.project}>
-              <input type="checkbox" name="configuration.user_chooses_workflow" value={@props.project.configuration?.user_chooses_workflow} onChange={handleInputChange.bind @props.project} />{' '}
+              {checked = @props.project.configuration?.user_chooses_workflow}
+              <input type="checkbox" name="configuration.user_chooses_workflow" checked={checked} value={checked} onChange={handleInputChange.bind @props.project} />{' '}
```

Comment:
I think these should be defaultChecked and defaultValue right? https://facebook.github.io/react/docs/forms.html